client/state/data-layer/wpcom-http/pipeline/remove-duplicate-gets/index.js

```diff
@@ -19,7 +17,7 @@ const debug = debugFactory( 'calypso:data-layer:remove-duplicate-gets' );
  * @module state/data-layer/wpcom-http/optimizations/remove-duplicate-gets
  */
 
-/** @type {Map} holds in-transit request keys */
+/** @type {Map<String,Array<Number, Object>>} holds in-transit request keys */
 const requestQueue = new Map();
```

Comment:
why is this called a queue again? its a map, not a queue

Reply:
it's not a queue data structure, but it's the _queue_ of network requests sitting out there. originally items disappeared when requests came back in, which meant that it resembled some kind of queue more like we'd imagine the queue at a bank.

now, it tracks the last time that a request came back (or is in progress) and resembles more of a ledger of sorts.

🤔 finding a new name might be good

Reply:
found one in 89b3b2d!